aggregator/src/datastore.rs

```diff
@@ -3619,6 +3726,56 @@ pub mod models {
     {
     }
 
+    /// An outstanding batch, which is a batch which has not yet started collection.
```

Comment:
Besides not yet having been collected, an `OutstandingBatch` can also still have reports assigned to it, right?

Reply:
Correct. It also only applies to `fixed-size` tasks. I noted both of these in the doc comment.